src/extensions/rich-text/code.ts

```diff
@@ -0,0 +1,10 @@
+import { Code } from '@tiptap/extension-code'
+
+const RichTextCode = Code.extend({
+    // Instead of not allowing other marks to coexist, allow everything so that
+    // inline code can be bolded, linked, etc
+    // See https://tiptap.dev/api/schema#excludes
+    excludes: Code.name,
```

Comment:
I believe the correct way to allow everything is with:

```suggestion
    excludes: '',
```

We don't really need to exclude `Code` from itself, no other mark does this. For instance, you don't see the `excludes` property in the [Bold mark](https://github.com/ueberdosis/tiptap/blob/main/packages/extension-bold/src/bold.ts). This is probaly handled internally, and there's no need to exclude the own mark.

Reply:
I was reading prose mirror's [documentation](https://prosemirror.net/docs/ref/#model.MarkSpec.excludes) which mentions that it defaults to marks of the same type, but I guess this behaviour is built in and not affected by the value here 👍